packages/grafana-data/src/types/datasource.ts

```diff
@@ -300,6 +300,7 @@ export interface QueryEditorProps<
    * Contains query response filtered by refId of QueryResultBase and possible query error
    */
   data?: PanelData;
+  range?: TimeRange;
   exploreId?: any;
```

Comment:
Range prop is always sent to Angular editors, but sometimes it is beneficial to have it in React Editors as well (e.g. fetching labels based on current time range).